db/migrate/20160628140841_fix_service_order_placed_at.rb

```diff
@@ -0,0 +1,12 @@
+class FixServiceOrderPlacedAt < ActiveRecord::Migration[5.0]
+  class ServiceOrder < ActiveRecord::Base; end
+
+  def up
+    update <<-SQL
```

Comment:
@Fryguy Should this be with using ActiveRecord ? Is there way how to generate this by using ActiveRecord or  we have to go thru each record and update it ?